.jenkins/infrastructure/build_azure_managed_images.Jenkinsfile

```diff
@@ -275,6 +284,34 @@ node(params.AGENTS_LABEL) {
 
         println("IMAGE_VERSION: ${image_version}\nIMAGE_ID: ${image_id}")
     }
+    stage("Install Azure CLI") {
+        retry(10) {
+            sh """
+                sleep 5
```

Comment:
Is it possible to call `installAzureCLI()` here instead?

Reply:
With how OpenEnclaveJenkinsLibrary defaults to master, it is not possible until `common.installAzureCLI()` (this PR) is merged into master.